compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/Types.java

```diff
@@ -6573,4 +6576,31 @@ public static class CommonAnalyzerData {
         int letCount = 0;
         boolean nonErrorLoggingCheck = false;
     }
+
+    public static byte[] convertToByteArray(String literalExpr) {
```

Comment:
Do these have to be static methods? The pattern we have followed (for most) is to call the methods via a types instance.

Reply:
Updated 9bc02e498c2314a3ddd5b48eb66ac7117ec63844